base/doc.dtx

```diff
@@ -148,8 +148,6 @@
 %
 % \newcommand\env{\texttt}
 % \newcommand\opt{\texttt}
-% \newcommand\cls{\texttt}
-% \newcommand\pkg{\texttt}
 % \newcommand\prg{\textsf}
```

Comment:
I guess I would probably still vote for doing \providecommand  (for all of them) rather than droppping some and keeping others.

There should be a at date change in the ProvidesPackage line (current version of doc is out in the wild)

Reply:
oh? I looked and it was only dated yeserday so I left it, I'll bump the date and add \providecommand